validate.py

```diff
@@ -157,7 +140,7 @@ def email_results(subject, results_text_string):
     if len(flags) == 0:
         flags.append("GREEN")
     flag_prefix = f"[{', '.join(flags)}] "
-    email_results(flag_prefix + job, results_text)
+    print(results_text)
```

Comment:
I think there's some benefit in using the job name and flag prefix as part of the text. So maybe replace the line you removed (160) with the following:

```
results_text = f"{flag_prefix} {job}\n\n"
```

Reply:
It'd also be good to get the `datetime.now(tz=pytz.UTC)` part in there. So maybe:

```
results_text = f"{flag_prefix} {job} for {datetime.now(tz=pytz.UTC)}\n\n" + results_text
```

Reply:
I was planning on just making this the subject line from Jenkins. Previously it was the subject line of the SMTP sent by this function. I think I can get the date in the subject line there too..

Reply:
You can't get the `flag_prefix` part once you're in Jenkins though, right?

Reply:
Oh hmm, the way this runs is it has this in a build and it sends the email in the post-build.

I think we'll have to have a way to pass this value between the build and the post-build. I'll look into this. Maybe we just have to set an environment variable. We might have to write the status to a file as an environment variable the read it into the post-build script. Maybe we'll just have to do that. I've never used it bu it looks like that's how [EnvInject](https://wiki.jenkins.io/display/JENKINS/EnvInject+Plugin) works. 

So just create a file in that OUT_DIR like:
unizin_validation_last.run> 
LAST_RUN_STATUS=GREEN/RED/YELLOW

This is safe because it's limited to one running job at a time.

Reply:
Ok I added this line back in